modules/serverless-logic-install-kn-workflow-plugin-cli.adoc

```diff
@@ -0,0 +1,81 @@
+// Module included in the following assemblies:
+//
+// * /serverless/install/install-serverless-operator.adoc
+
+:_content-type: PROCEDURE
+[id="serverless-logic-install-kn-workflow-plugin-cli_{context}"]
+= Installing the {ServerlessLogicOperatorName} using the Knative workflow plugin from the CLI
+
+OpenShift Serverless Logic Workflow provides a plugin named `kn-plugin-workflow` for the Knative CLI, enabling you to set up a local workflow project using the command line.
+
+.Prerequisites
+
+* {ServerlessLogicOperatorName} is installed on your cluster.
+* You have installed the Knative (`kn`) CLI.
+* You have created a project or have access to a project with the appropriate roles and permissions to create applications and other workloads in {ocp-product-title}.
+
+.Procedure
+
+. Download the latest binary file.
```

Comment:
Maybe make it more clear that the sub-steps show how to do this. I didn't realize that when I first read it.

```suggestion
. Download the latest `kn-workflow` binary file by using the following steps:
```